wrappers/kn/src/nativeMain/kotlin/zxingcpp/ImageView.kt

```diff
@@ -0,0 +1,52 @@
+package zxingcpp
+
+import cnames.structs.ZXing_ImageView
+import kotlinx.cinterop.*
+import zxingcpp.cinterop.*
+import kotlin.experimental.ExperimentalNativeApi
+import kotlin.native.ref.createCleaner
+
+@OptIn(ExperimentalForeignApi::class)
+class ImageView(
+	val data: ByteArray,
+	val width: Int,
+	val height: Int,
+	val format: ImageFormat,
+	val rowStride: Int = 0,
+	val pixStride: Int = 0,
+) {
+	private val pinnedData = data.pin()
+	val cValue: CPointer<ZXing_ImageView>? =
+		ZXing_ImageView_new(
```

Comment:
Please use the `ZXing_ImageView_new_checked` version here and pass `data.size` as the second parameter.